Automattic/jetpack
# Add/calculate scheduled shares on activation

This PR is related to the task mentioned below. The changes introduce an API call that goes to WPCOM to schedule an async job which recalculates the number of scheduled posts. It is called on every plugin activation.
Patch that introduces new WPCOM endpoint: D85220-code

#### Changes proposed in this Pull Request:

* Added function that uses `wpcom_json_api_request_as_blog` to call the the WPCOM endpoint
* Connected this function into the `plugin_activation` hook

#### Jetpack product discussion

1202657984845794-as-1202660718919317

#### Does this pull request change what data or activity we track or use?

No.

#### Testing instructions:






* As long as the endpoint changes (D85220-code) are not merged, pull that patch and use sandbox as backend
* Sandbox the `async_count_scheduled_shares_sandboxed` job
* The job should get called every time the plugin is activated
